src/main/java/pl/futurecollars/invoicing/utils/FilesService.java

```diff
@@ -9,22 +9,16 @@
 
 public class FilesService {
 
-  public void createRepository(Path idPath, Path databasePath) throws IOException {
-    createIdPathRepository(idPath);
-    createDatabasePathRepository(databasePath);
+  public void initDatabase(Path idPath, Path databasePath) throws IOException {
+    createFile(idPath);
+    createFile(databasePath);
   }
 
-  private void createIdPathRepository(Path idPath) throws IOException {
+  private void createFile(Path idPath) throws IOException {
```

Comment:
1. nazwa parametru nie pasuje do kontekstu metody
2. ta metoda nie potrzebuje obiektu Path, lecz String (widać to w momencie, gdy metoda ta jedyne co robi na obiekcie Path to zamienia go na String), więc mogła by od razu String przyjmować.